azavea/tilegarden
# Remove transpiler + a few small improvements

## Overview

Applies some of the changes noted in #129 that were done in the https://github.com/azavea/pfb-network-connectivity and are worth bringing back into this version.  Most notably, removing the transpilation step.  Since Node still doesn't support ES6-style modules, this meant downgrading all the imports and utility modules to the CommonJS-style `require`/`module.exports =` setup, but that seems like a reasonable price to pay for the reduced complexity.

This also applies some small changes:
- Removes the `util/fs-promise.js` helper, using the built-in `util.promisify` instead
- Switches to an Azavea fork of `claudia-local-api`--currently identical to @mattdelsordo's fork but there's a decent chance we'll need more tweaks, and it makes sense to have it in the fold anyway.
- Makes it return a specific error when there's no default map config available

## Testing Instructions

- run `scripts/update` and `scripts/server`
- open your demo pages to confirm that everything works as before

Connects #129.
